### DISCUSSION TITLE
Local development

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [a-tokyo/react-apple-signin-auth#71] [Q&A]
Hi! Do you know if there's a **COMPLETE** guide on how to test the apple sign in locally? I've tried to follow this one -> https://next-auth.js.org/providers/apple and other similar who says to update the host file but for some reason, none of them work.

**Comment by a-tokyo** [comment]
Apple doesn't allow localhost. I am not aware of any method to test locally. I usually spin up a sandbox/staging environment online and test on it.
